src/main/webapp/scripts/map.js

```diff
@@ -172,11 +172,11 @@ function placeMarkerAndPanTo(latLng, map) {
   const marker = new google.maps.Marker({
     position: latLng,
     map: map,
-    icon: greyIcon
+    //icon: greyIcon
   });
   /* Recenter the Map */
   map.panTo(latLng);
-  togglePopup(latLng, marker);
+  togglePopup(latLng);
```

Comment:
I am not sure but togglePopup function below does not take any parameter

Reply:
Good point - this was updated in a more up-to-date branch but I'll update it here too